packages/cetlib/package.py

```diff
@@ -40,15 +40,16 @@ class Cetlib(CMakePackage):
 
     depends_on("boost+regex+program_options+filesystem+system+test")
     depends_on("cetlib-except")
-    depends_on("hep-concurrency")
+    depends_on("hep-concurrency", when="@3.19:", type=("build", "test"))
+    depends_on("hep-concurrency", when="@:3.18.99")
     depends_on("openssl")
     depends_on("perl", type=("build", "run"))
     depends_on("sqlite@3.8.2:")
     depends_on("tbb")
```

Comment:
`tbb` should also be a build-only dependency for tests

Reply:
Actually, come to think of it.  I wonder if the `tbb` dependency should just be removed since it will be provided by `hep-concurrency`....?

Reply:
See 0f0396c.